pkg/support/dump_others.go

```diff
@@ -60,21 +65,58 @@ func (d *agentDumper) dumpIPTables(basedir string) error {
 }
 
 func (d *agentDumper) dumpIPToolInfo(basedir string) error {
-	dump := func(name string) error {
-		output, err := d.executor.Command("ip", name).CombinedOutput()
+	dump := func(args ...string) error {
+		output, err := d.executor.Command("ip", args...).CombinedOutput()
 		if err != nil {
-			return fmt.Errorf("error when dumping %s: %w", name, err)
+			return fmt.Errorf("error when dumping %s: %w", strings.Join(args, " "), err)
 		}
-		return writeFile(d.fs, filepath.Join(basedir, name), name, output)
+		return writeFile(d.fs, filepath.Join(basedir, args[0]), args[0], output)
+	}
+	commands := [][]string{
+		{"link"},
+		{"address"},
+		{"rule", "show"},
+		{"route", "show", "table", "all"},
 	}
-	for _, item := range []string{"route", "link", "address"} {
-		if err := dump(item); err != nil {
+	for _, cmd := range commands {
+		if err := dump(cmd...); err != nil {
 			return err
 		}
 	}
 	return nil
 }
 
+func (d *agentDumper) dumpInterfaceConfigs(basedir string) error {
```

Comment:
can we add a unit test for this function? And ideally for `dumpIPToolInfo` as well.